charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl

```diff
@@ -21,6 +21,15 @@ acme:
   privateKey: |
 {{ .Values.certificateConfig.defaultIssuer.acme.privateKey | trim | indent 4 }}
   {{- end }}
+{{- if .Values.customConfig }}
+custom:
```

Comment:
What do you think about reusing the `ACME` structure here? `custom` sounds very generic and the fields below are actually ACME related.